awx/ui/test/e2e/tests/test-searchez.js

```diff
@@ -0,0 +1,125 @@
+import { range } from 'lodash';
+
```

Comment:
this served its purpose well for sanity checking when moving the data transformation logic out of the search controller and into the service but I'm to need going to come back and add unit tests after we get everything else finished.